sklearnex/preview/ensemble/__init__.py

```diff
@@ -15,8 +15,12 @@
 # limitations under the License.
 # ===============================================================================
 
-from .extra_trees import ExtraTreesClassifier, ExtraTreesRegressor
-from .forest import RandomForestClassifier, RandomForestRegressor
+from .extra_trees import (
+    ExtraTreesClassifier,
+    ExtraTreesRegressor,
+    RandomForestClassifier,
+    RandomForestRegressor,
+)

```

Comment:
`extra_trees` module should be renamed to a common name for ExtraTrees and RandomForest